src/app/app.config.ts

```diff
@@ -1,116 +1,133 @@
+      'x_editor_priority': 1
     },
-    "accelerator_experiments.items.properties.curated_relation": {
-      "x_editor_hidden": true
+    'accelerator_experiments.items.properties.curated_relation': {
+      'x_editor_hidden': true
     },
-    "accelerator_experiments.items.properties.record": {
-      "x_editor_hidden": true
+    'accelerator_experiments.items.properties.record': {
+      'x_editor_hidden': true
     },
-    "arxiv_eprints.items.properties.value": {
-      "x_editor_priority": 1
+    'arxiv_eprints.items.properties.value': {
+      'x_editor_priority': 1
     },
-    "authors.items.properties.affiliations.items.properties.curated_relation": {
-      "x_editor_hidden": true
+    'authors.items.properties.affiliations.items.properties.curated_relation': {
+      'x_editor_hidden': true
     },
-    "authors.items.properties.affiliations.items.properties.record": {
-      "x_editor_hidden": true
+    'authors.items.properties.affiliations.items.properties.record': {
+      'x_editor_hidden': true
     },
-    "authors.items.properties.affiliations.items.properties.value": {
-      "x_editor_priority": 1
+    'authors.items.properties.affiliations.items.properties.value': {
+      'x_editor_priority': 1
     },
-    "authors.items.properties.full_name": {
-      "x_editor_priority": 1
+    'authors.items.properties.full_name': {
+      'x_editor_priority': 1
     },
-    "authors.items.properties.affiliations": {
-      "x_editor_priority": 2
+    'authors.items.properties.affiliations': {
+      'x_editor_priority': 2
     },
-    "authors.items.properties.curated_relation": {
-      "x_editor_hidden": true
+    'authors.items.properties.curated_relation': {
+      'x_editor_hidden': true
     },
-    "authors.items.properties.uuid": {
-      "x_editor_hidden": true
+    'authors.items.properties.uuid': {
+      'x_editor_hidden': true
     },
-    "authors.items.properties.record": {
-      "x_editor_hidden": true
+    'authors.items.properties.record': {
+      'x_editor_hidden': true
     },
-    "collaboration.items.properties.record": {
-      "x_editor_hidden": true
+    'collaboration.items.properties.record': {
+      'x_editor_hidden': true
     },
-    "collaboration.items.properties.value": {
-      "x_editor_priority": 1
+    'collaboration.items.properties.value': {
+      'x_editor_priority': 1
     },
-    "dois.items.properties.value": {
-      "x_editor_priority": 1
+    'control_number': {
+      'x_editor_hidden': true
     },
-    "external_system_numbers.items.properties.value": {
-      "x_editor_priority": 1
+    'dois.items.properties.value': {
+      'x_editor_priority': 1
     },
-    "field_categories.items.properties.term": {
-		  "x_editor_priority": 1
+    'external_system_numbers.items.properties.value': {
+      'x_editor_priority': 1
     },
-    "hidden_notes.items.properties.value": {
-		  "x_editor_priority": 1
+   // FIX: Issue with the anyOf field 
```

Comment:
You can try now with latest version of `ng2-json-editor` since anyOf is supported.